test/productroutes.test.js

```diff
@@ -0,0 +1,262 @@
+const expect = require("chai").expect;
+const request = require("supertest");
+
+const router = require("../server/");
+//const agent = request.agent(app);
+
+const db = require("../server/database/connection.js");
+const Product = require("../server/database/models/product.js");
+
+/**
+ *
+ * Article Route Tests
+ *
+ * Do these after you finish the Article Model tests
+ *
+ */
+describe("Products Route:", () => {
+  /**
+   * First we clear the database before beginning each run
+   */
+  before(() => {
+    return db.sync({ force: true });
+  });
+
+  /**
+   * Also, we empty the tables after each spec
+   */
+  afterEach(() => {
+    // return Promise.all([Product.truncate({ cascade: true })]);
+  });
+
+  describe("GET /api/products", () => {
+    it("responds with an array via JSON", async () => {
+      const res = await agent
+        .get("/api/articles")
+        .expect("Content-Type", /json/)
+        .expect(200);
+
+      // res.body is the JSON return object
+      expect(res.body).to.be.an.instanceOf(Array);
+      expect(res.body).to.have.length(0);
+    });
+
+    /**
+     * Problem 2
+     * Save an article in the database using our model and then retrieve it
+     * using the GET /articles route
+     *
+     */
+    it("returns an article if there is one in the DB", async () => {
+      await Article.create({
```

Comment:
I'm not really sure what this is testing, I don't think there is an articles model